tasks/master.yml

```diff
@@ -1,15 +1,13 @@
 ---
+
 - name: Install the master server
-  shell: ipa-server-install -r {{ ansible_domain | upper }} -n {{ ansible_domain }} -p {{ directory_admin_password }}  -U -a {{ kerberos_admin_password }} --ip-address {{ ansible_default_ipv4.address }}
+  shell: >
+    ipa-server-install -U \
+      -r {{ freeipa_domain | upper }} -n {{ freeipa_domain }} \
+      -p {{ directory_admin_password | quote }} -a {{ kerberos_admin_password | quote }} \
+      --hostname {{ inventory_hostname }} --ip-address {{ frontnet_addr }} \
+      -N --setup-dns --auto-forwarders --forward-policy only
```

Comment:
Since we're forking this from someone else's module I didn't bother with other comments on styling.  I've been trying to mimic the styling of the original author when able so if you feel like the above comment is over the line you can ignore.  That said for our stuff it needs to be that way.